projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -345,6 +345,12 @@ export class IgxGridBaseDirective extends DisplayDensityBase implements
     @Output()
     public filteringExpressionsTreeChange = new EventEmitter<IFilteringExpressionsTree>();
 
+    /**
+     *@hidden
+     */
+    @Output()
+    public advancedFilteringExpressionsTreeChange = new EventEmitter<IFilteringExpressionsTree>();
```

Comment:
We should remove the **@hidden** tag and document this emitter as a **new** feature.

Reply:
Done.

Reply:
Done.